src/soft-play.js

```diff
@@ -6,9 +6,63 @@ let children = 0
 // the adults and children variables defined above.
 // Start with the occupancy function.
 
+// first we initialize the variables that keep track of the adults and kids
+let numAdults = 0;
+let numChildren = 0;
+
+// here is what the object of the softballCenter looks like
+let softballCenter = {
+  "adults": numAdults,
+  "children": numChildren
+}
+
+// here's the tracker that tracks the total
+let softballCenterTotal = {
+  "adults": 0,
+  "children": 0,
+}
+
+// function that basically only prints out the current state of the softball-Center-object
+const occupancy = (() => softballCenter)
+const total = (() => softballCenterTotal)
+
+//adding shit so i can easily reset them – not necessarily used
+const resetOccupancy = (() => {
+  softballCenter.adults = 0,
+  softballCenter.children = 0
+})
```

Comment:
Good use of arrow functions but you don't need to wrap the whole thing in parentheses:

```js
const occupancy = () => softballCenter
const total = () => softballCenterTotal

const resetOccupancy = () => {
  softballCenter.adults = 0,
  softballCenter.children = 0
}
```

Reply:
Thank you – I'm concious of the problem that I'm shaky on the syntax when adding too many components together (arrow functions, unnamed functions directly feeding into the variable, in other cases array methods too) – so rn I'm throwing parantheses and curly brakets in there to see whether it'll run then :D I'm working on it!